doc/integrator/checker.rst

```diff
@@ -50,6 +50,29 @@ The checker use the following configuration structure:
         checker:
             forward_headers: ['Cookie', 'Authorisation']
 
+.. note::
+
+    The checker assumes that it can access the c2cgeoportal services via ``http://localhost``.
+    If this is not allowed on your server, you can override this behaviour as follows.
+    In the configuration file (see above), add the following:
+
+    .. code:: yaml
+
+        vars:
+            checker:
+                rewrite_as_http_localhost: False
+
+    Now, in your project YAML configuration, instead of defining the ``checker_path``,
+    define a `checker_url` with the full URL to be used, for example:
```

Comment:
You should use double back quote in restructured text...

Yous not specify the file name (`project.yaml.mako`)?

Reply:
Yes thanks, I will change that